pyproject.toml

```
@@ -33,12 +33,13 @@ classifiers = [
   "Topic :: Software Development :: Libraries",
   "Typing :: Typed",
 ]
-requires-python = ">= 3.11, < 4"
+requires-python = ">= 3.10, < 4"
 dependencies = [
   "betterproto[compiler] == 2.0.0b6",
   "grpcio-tools >= 1.59.0, < 2",
   "setuptools >= 67.6.0",
   "typing-extensions >= 4.5.0, < 5",
+  "tomli >= 2.0.1; python_version < '3.11'",
```

Sorry, can you also limit the maximum version here, to avoid unexpected breakage if `tomli` 3 is released with breaking changes? We use dependabot to bump dependencies when new versions come out, so we can widen the restriction once we know it still works with new `tomli` major versions.

```suggestion
  "tomli >= 2.0.1, < 3; python_version < '3.11'",
```